config/autoload/global.php

```diff
@@ -0,0 +1,17 @@
+<?php
+return [
+    'zf-content-negotiation' => [
+        'selectors' => [
+            'Foo' => [
+                'Zend\\View\\Model\\JsonModel' => [
```

Comment:
Ironically, this entry shouldn't be there at all; it was a test selector I'd added when testing the UI. :smile: